packages/okta-angular/test/e2e/harness/scripts/prebuild.js

```diff
@@ -12,6 +12,20 @@
 
 const fs = require('fs');
 const path = require('path');
+const shell = require('shelljs');
+
+// Link the local module. We are not using yarn link because we do not want to modify the global state
+const MODULE_NAME = '@okta/okta-angular';
+const SRC_DIR = path.resolve(__dirname, '..', '..', '..', '..', 'dist');
+const DEST_LINK = path.resolve(__dirname, '..', 'node_modules', MODULE_NAME);
+const DEST_DIR = path.resolve(DEST_LINK, '..');
+
+shell.echo(`Linking module: ${MODULE_NAME} from ${SRC_DIR}`);
+shell.mkdir(`-p`, `${SRC_DIR}`);
+shell.mkdir(`-p`, `${DEST_DIR}`);
+shell.exec(`ln -sF ${SRC_DIR} ${DEST_LINK}`);
+
+// Generate environment.ts from the OIDC config
 const Config = require('../../../../../../.oidc.config.js');
```

Comment:
You didn't add this line, but I need someone to acknowledge the tears I weep each time I see it.

Reply:
Yeah its horrid. The only consolation is that this is an E2E test harness, so it is not offered as a sample or "best practice". One possible solution would be to put the config its own module so we could doe something like  `require('shared-config')`, but that will not work well in this particular lerna monorepo. We have the modules explicitly isolated from each other, so it would not link the way it would in a yarn workspace.